packages/admin-panel/src/VizBuilderApp/components/Panel.js

```diff
@@ -76,6 +76,17 @@ const PanelTabPanel = styled.div`
   }
 `;
 
+const CustomReportName = styled.span`
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+
+  > div {
+    width: 100%;
+    height: 100%;
+  }
+`;
```

Comment:
I am no css craftsperson 😢 You can see the ticket for an example. But given this should be for internal use only I don't mind it if it looks a bit bad

Reply:
Replace `CustomReportName` to div will do the work, don't need to attach any css.

Reply:
Ah thanks @billli0 !